src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -403,6 +461,7 @@ import Tooltip from '@/components/ui/Tooltip.vue';
 import UserPicture from '@/components/ui/UserPicture.vue';
 import EditPictureModal from './EditPictureModal.vue';
 import Fetcher from '@/assets/js/fetcher';
+import Fluent from '@/components/Fluent.vue';
 
 // TODO: Why do we need failoverOnError here?
 const fetcher = new Fetcher({ failoverOn: [302], failoverOnError: true });
```

Comment:
I don't think we do? We also should move it even if we do to be out of the component.

Reply:
Is this referencing `TODO: Why do we need failoverOnError here?` Florian removed this in a separate PR: https://github.com/mozilla-iam/dino-park-front-end/pull/510/files#diff-65e12fed037aa326de35fbf8f19bb04fL407

Reply:
Wonderful!